### PR TITLE
{chem}[gompi/2022a] ORCA v5.0.4

### PR DESCRIPTION
**Pull request by bedroge**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 22.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.10.6
See https://gist.github.com/24c9b7b9b5c1db73890277ced7e5eea2 for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
pg-interactive.hpc.rug.nl - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/5fc6e7d6b9b363c195a88a0ca13fd84c for a full test report.

File: easybuild/easyconfigs/o/ORCA/ORCA-5.0.4-gompi-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ORCA is a flexible, efficient and easy-to-use general purpose tool for quantum
+chemistry with specific emphasis on spectroscopic properties of open-shell
+molecules. It features a wide variety of standard quantum chemical methods
+ranging from semiempirical methods to DFT to single- and multireference
+correlated ab initio methods. It can also treat environmental and relativistic
+effects."""
+
+toolchain = {'name': 'gompi', 'version': '2022a'}
+
+# Shared build of ORCA: download from https://orcaforum.kofo.mpg.de
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/o/ORCA/ORCA-5.0.4-gompi-2022a.eb:15]: We could make use of `download_instructions` here, maybe something like:

```suggestion
download_instructions = "Shared build of ORCA: download from https://orcaforum.kofo.mpg.de"
```

Comment (bedroge) [reply easybuild/easyconfigs/o/ORCA/ORCA-5.0.4-gompi-2022a.eb]: Done, thanks!

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bwd-rockylinux-91 - Linux Rocky Linux 9.1 (Blue Onyx), x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.9.14
See https://gist.github.com/1869cceee126480dabf3ae09ef8f06cf for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
zen2-ubuntu-2210 - Linux Ubuntu 22.10 (Kinetic Kudu), x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.10.7
See https://gist.github.com/0b20a1fb3d491ebbb0ed05d71dc1c183 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
skl-opensuse-154 - Linux openSUSE Leap 15.4, x86_64, Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz (skylake), Python 3.6.15
See https://gist.github.com/7a9b3a466464fa8104433b4151c67d06 for a full test report.

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Comment by SebastianAchilles** [comment]
Going in, thanks @bedroge!

**Pull request merged**